stdlib/websub/src/main/ballerina/src/websub/commons.bal

```diff
@@ -623,12 +604,11 @@ public type Hub object {
 
         if (stopResult is error) {
             if (stopHubServiceResult is error) {
-                error[] causes = [stopResult, stopHubServiceResult];
-                return error(WEBSUB_ERROR_CODE, causes = causes);
+                return WebSubError("Couldn't stop the started up Ballerina WebSub Hub", stopHubServiceResult);
```

Comment:
now can't we pass an array of causes like before?

Reply:
Yes, we can't pass an array of causes now.